closed/src/java.base/share/classes/com/sun/crypto/provider/NativeGaloisCounterMode.java

```diff
@@ -343,40 +359,48 @@ int encrypt(byte[] in, int inOfs, int len) {
      */
     int encryptFinal(byte[] in, int inOfs, int len, byte[] out, int outOfs)
         throws IllegalBlockSizeException, ShortBufferException {
+        synchronized (this) {
+            byte[] keyCopy;
+            byte[] ivCopy;
+            byte[] aad;
+            int tagLenBytes_local = this.tagLenBytes;
```

Comment:
Please use a consistent naming convention for local variables (no others here use `_`).

Reply:
I misinterpreted [this](https://github.com/ibmruntimes/openj9-openjdk-jdk16/pull/66#discussion_r670512976) comment thinking that you are suggesting I move the 4 of variables (`keyCopy`, `ivCopy`, `aad`, and `tagLenBytes_local`) to the synchronized block. But I am now realizing you meant that I only move the initialization of the `tagLenBytes_local`. So I moved all 4 definitions outside the synchronization block and renamed `tagLenBytes_local` to `localTagLenBytes`.